assets/prometheus-operator-user-workload/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/name: prometheus-operator
+    app.kubernetes.io/version: v0.33.0
+  name: prometheus-operator
+  namespace: openshift-user-workload-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: controller
+      app.kubernetes.io/name: prometheus-operator
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: controller
+        app.kubernetes.io/name: prometheus-operator
+        app.kubernetes.io/version: v0.33.0
+    spec:
+      containers:
+      - args:
+        - --kubelet-service=kube-system/kubelet
+        - --logtostderr=true
+        - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
+        - --prometheus-config-reloader=quay.io/coreos/prometheus-config-reloader:v0.33.0
+        - --deny-namespaces=openshift-monitoring
```

Comment:
I don't know why I thought it would be its own namespace on the deny list as well? 🤔

Reply:
indeed, this is just an example setting, it is being overwritten by cluster-monitoring-operator at runtime.

Reply:
(we use that "example setting" idiom in the other operator too)